src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/serializer/CQLBindValues.java

```diff
@@ -30,18 +30,18 @@ public static Set<String> getSetValue(Set<JsonPath> from) {
     return from.stream().map(val -> val.toString()).collect(Collectors.toSet());
   }
 
-  public static Set<String> getStringSetValue(Set<String> from) {
+  public static Set<String> getStringSetValue(Set<?> from) {
```

Comment:
Changed so it can be called for Sets of other types: no point to convert from `Set<String>` to `Set<String>` :)

Reply:
Do we have any other type need it?

Reply:
Looks like there is alternative `getSetValue()`; I can unify the two I think.

Reply:
Reverted signature change but changed code to create simple copy. Not convinced copy is really necessary but that can be changed later on.